.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 		es6: true
 	},
 	parserOptions: {
-		ecmaVersion: 2018
+		ecmaVersion: 2020
```

Comment:
Why is this needed?

Our code should stay in lower es style to be compatible with node.js 10.

Maybe use `BigInt("123")` instead of `123n` in code to avoid that.

Reply:
Done :+1: